remote/lib/settings.dart

```diff
@@ -0,0 +1,232 @@
+
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'theme_model.dart';
+
+class SettingsPage extends StatefulWidget {
+  const SettingsPage({Key? key}) : super(key: key);
+
+  @override
+ State<SettingsPage> createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  @override
+  Widget build(BuildContext context) {
+   
+    
+  return Consumer(
+    builder:(context,ThemeModel themeNotifier,child){
+    
+    return Scaffold(
+      appBar: AppBar(title: const Center(child: Text('Settings')),
+      ),
+      
+      body: Container(
+        padding: const EdgeInsets.only(left: 16, top: 25, right: 16),
+        child: ListView(
+          children: [
+         
+            Row(
+           
+              children: const [
+                
+               Text(
+                  "General",
+                  style: TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
+                ),
+              ],
+            ),
+            const Divider(
+              height: 15,
+              thickness: 2,
+            ),
+            const SizedBox(
+              height: 10,
+            ),
+           
+        Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Text(
+          'Change theme',
+          style: TextStyle(
+              fontSize: 18,
+              fontWeight: FontWeight.w500,
+              color: Colors.grey[600]),
+        ),
+        Transform.scale(
+            scale: 0.7,
+           child:IconButton(onPressed:  (){ 
+          themeNotifier.isDark
+          ? themeNotifier.isDark=false:themeNotifier.isDark=true;
+        }, icon: Icon(themeNotifier.isDark?Icons.wb_sunny:Icons.nightlight_round)))
+      ],
+    ),
+
+            //buildAccountOptionRow(context, "Change theme"),
+            buildNotificationOptionRow("Wake lock", true),
+            buildNotificationOptionRow("Confirm before streaming", true),
```

Comment:
I guess we could remove these items we don't use, but we can do that in another PR.